shell.nix

```diff
@@ -11,7 +11,6 @@ let
     git
     jq
     niv
-    nix-linter
```

Comment:
Temporary (and perhaps unnecessary) but I was getting

```
error: Package ‘nix-linter-0.2.0.4’ in /nix/store/hi1s6s43602ypv0vqi8gy36g01gw6l6s-nixpkgs-src/pkgs/development/tools/analysis/nix-linter/default.nix:23 is marked as broken, refusing to evaluate.
```

Reply:
This is fine, we'll move to `statix` + `deadnix`.